tests/zkevm/test_worst_bytecode.py

```diff
@@ -68,6 +68,7 @@ def test_worst_bytecode_single_opcode(
     # We use 100G gas limit to be able to deploy a large number of contracts in a single block,
     # avoiding bloating the number of preparing blocks in the test.
     env = Environment(gas_limit=100_000_000_000)
```

Comment:
Running locally, it seems like the contract deploy tx is running out of gas, hence the CI failure.

This change seems to do the trick:
```suggestion
    env = Environment(gas_limit=1_000_000_000_000)
```

Reply:
Using geth for filling doesn't fail and the CI run before the last two commits (which are adding comments and renaming a parameter) passed without problem.

I'll bump this limit and see what happens :man_shrugging: